README.md

```diff
@@ -3,15 +3,29 @@
 See documentation [here](doc/00-overview.md)
 
 ``` yaml
+
+debug-output: $(output-folder)/_az_debug
+
 use-extension:
-  "@autorest/python": "5.0.0-dev.20200211.1"
-  "@autorest/clicommon": "0.3.5"
+  "@autorest/clicommon": "c:/git/autorest.clicommon"

```

Comment:
shouldn't be here

Reply:
good catch. this PR is just for showing some code. [Ignore this] in the title. :)